docs/running-on-kubernetes.md

```diff
@@ -0,0 +1,512 @@
+---
+layout: global
+title: Running Spark on Kubernetes
+---
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+Spark can run on clusters managed by [Kubernetes](https://kubernetes.io). This feature is currently experimental.
+
+# Prerequisites
+
+* A spark distribution with Kubernetes.
```

Comment:
Add a reference to releases: `([releases](https://github.com/apache-spark-on-k8s/spark/releases))`.

Reply:
This is for upstream, so, it shouldn't link back to our fork - maybe I'll add "Spark 2.3 and above" to clarify?

Reply:
Sounds good to me.